hardhat.config.js

```diff
@@ -8,14 +8,20 @@ const MAINNET_RPC_URL = process.env.MAINNET_RPC_URL
 const POLYGON_MAINNET_RPC_URL = process.env.POLYGON_MAINNET_RPC_URL
 const MUMBAI_RPC_URL = process.env.MUMBAI_RPC_URL
 const GOERLI_RPC_URL = process.env.GOERLI_RPC_URL
+const SEPOLIA_RPC_URL = process.env.SEPOLIA_RPC_URL
 
 // Ensure one of the RPC endpoints has been set
-let setRpcUrlCount = 0
-if (!MAINNET_RPC_URL && !POLYGON_MAINNET_RPC_URL && !MUMBAI_RPC_URL && !GOERLI_RPC_URL) {
-  throw Error(
-    "One of the following environment variables must be set: MAINNET_RPC_URL, GOERLI_RPC_URL, POLYGON_MAINNET_RPC_URL, or MUMBAI_RPC_URL"
-  )
+const selectedRPCs = [
+  MAINNET_RPC_URL, 
+  POLYGON_MAINNET_RPC_URL,
+  MUMBAI_RPC_URL,
+  GOERLI_RPC_URL,
+  SEPOLIA_RPC_URL
+].filter(v => !!v)
+if (selectedRPCs.length !== 1) {
+  throw Error("A single *_RPC_URL env variable must be set, found: " + selectedRPCs.join(","))
```

Comment:
I still don't get why we need to have exactly one of them set... It's more convenient to set them all. I'd even like to have per-network PRIVATE_KEYs set, so I can switch between networks easily.

Reply:
Correct.  We no longer need only 1 set.  I fixed that.  The previous reason was to figure out which chain to fork for the simulator, but this was fixed by using the following statement:
`url: MAINNET_RPC_URL ?? POLYGON_MAINNET_RPC_URL ?? GOERLI_RPC_URL ?? MUMBAI_RPC_URL`

However, the statement above will not work correctly if we set the unset RPC URLs to `"UNSET"`